README.md

```diff
@@ -118,12 +118,14 @@ If you do not provide a configuration file, the exporter creates one from the pr
 
 #### Rule configuration
 
-| Key                | Description                                                                                                   |
-|--------------------|---------------------------------------------------------------------------------------------------------------|
-| rules.clusters     | List of clusters to fetch metrics from                                                                        |
-| rules.labels       | Labels to exposed to Prometheus and group by in the query                                                     |
-| rules.topics       | Optional list of topics to filter the metrics                                                                 |
-| rules.metrics      | List of metrics to gather                                                                                     |
+| Key                         | Description                                                                                                   |
+|-----------------------------|---------------------------------------------------------------------------------------------------------------|
+| rules.clusters              | List of clusters to fetch metrics from                                                                        |
+| rules.labels                | Labels to exposed to Prometheus and group by in the query                                                     |
+| rules.topics                | Optional list of topics to filter the metrics                                                                 |
+| rules.excludedTopics        | Optional list of topics to exclude from the query                                                             |
+| rules.excludedTopicsRegex   | Optional regex to filter the result of the query                                                              |
```

Comment:
I am just thinking out loud, but I think, for consistency purposes, we should provide the same options for including and excluding topics. I think we should either:

-  Add a `topicsRegex` option; or
- Just have a `topics` and `excludeTopics` options that are both Regex 

What do you think @abraham-leal ?

Reply:
Ohh, I didn't realize that there is no regex available on the metrics API, and thus the regex filter will be done locally on the exporter node :/

We should find a way to make it clear to the users that, this option will not be translated to a Metrics API query and will actually be performed locally. Maybe by renaming by something like: `rules.filterLocallyTopicsRegex` (or something like that)

Reply:
Good point. What do you think of `rules.topicMetricsLocalFilterRegex`? I'm horrible at naming, so don't laugh haha.

Reply:
`topicMetricsLocalFilterRegex` looks like a good name ;)
I am terrible at this as well :sweat_smile: